### PR TITLE
Bishoy swap any js alert with req flash

### PR DESCRIPTION
**Pull request by BishoyHanna91**
Swaped out any javascript alert() messages with the new req.flash() messages in the user page 
Removed the badgeinSuccess.njk page

### CODE REVIEW

File: routes/index.js [hunk 1]
```diff
@@ -254,42 +243,48 @@ module.exports = function (passport) {
...
     }
-    var BagdeID = req.params.badge;
+    var badgeID = req.params.badge;
     var StationID = req.params.station;
-    dbAPI.removePrivileges(BagdeID, StationID).then(function (result) {
+    dbAPI.removePrivileges(badgeID, StationID).then(function (result) {
       console.log(result);
     });
-    res.redirect('/userManagement/' + BagdeID);
+    req.flash('success', 'This user is revoked from this machine');
```
Comment (danieleynis) [inline routes/index.js]: Change this message to be "The users access to the selected station has been revoked"

File: routes/index.js [hunk 2]
```diff
@@ -314,7 +309,9 @@ module.exports = function (passport) {
           }
         });
       }
-      res.redirect('/userManagement/' + BagdeID);
+      req.flash('success', 'This user is promoted successfully');
```
Comment (danieleynis) [inline routes/index.js]: Change this message to "The user has been successfully promoted to Manager status". We want to be pretty verbose with our messages so there is no room for confusion.

File: routes/index.js [hunk 3]
```diff
@@ -254,42 +243,48 @@ module.exports = function (passport) {
...
     }
-    var BagdeID = req.params.badge;
+    var badgeID = req.params.badge;
     var StationID = req.params.station;
-    dbAPI.grantPrivileges(BagdeID, StationID).then(function (result) {
+    dbAPI.grantPrivileges(badgeID, StationID).then(function (result) {
       console.log(result);
     });
-    res.redirect('/userManagement/' + BagdeID);
+    req.flash('success', 'This user is granted to this machine');
```
Comment (danieleynis) [inline routes/index.js]: Change this to "The user has been successfully granted access to the selected station"

File: routes/index.js [hunk 4]
```diff
@@ -341,7 +338,9 @@ module.exports = function (passport) {
           }
         });
       }
-      res.redirect('/userManagement/' + BagdeID);
+      req.flash('success', 'This user is demoted successfully');
```
Comment (danieleynis) [inline routes/index.js]: Change this to "The Manager has been successfully demoted to User status"

File: routes/index.js [hunk 5]
```diff
@@ -351,24 +350,24 @@ module.exports = function (passport) {
...
             console.log("Error");
           }
           else {
             console.log("user Badged out successfully");
-            
+
           }
         });
       }
       req.flash('success', 'This user badged out the lab successfully');
```
Comment (danieleynis) [inline routes/index.js]: Change this to "The user has been successfully badged out of the lab"

**Comment by BishoyHanna91** [comment]
I made the changes as u requested

File: routes/index.js [hunk 6]
```diff
@@ -254,42 +243,48 @@ module.exports = function (passport) {
...
+    dbAPI.modifyUser(badgeID, { confirmation: true }).then(function (result) {
       if (result == undefined) {
         console.log("Error");
       }
       else {
         console.log("confirmation changed successfully");
       }
     });
-    res.redirect('/userManagement/' + BagdeID);
+    req.flash('success', 'This user is cofirmed successfully');
```
Comment (danieleynis) [inline routes/index.js]: Type here: should be "confirmed"

Comment (BishoyHanna91) [reply routes/index.js]: instead of success ?

File: routes/index.js [hunk 7]
```diff
@@ -246,6 +233,8 @@ module.exports = function (passport) {
     dbAPI.deleteUser(DeletedBadge).then(function (result) {
       console.log("user is deleted successfully");
     });
+    req.flash('success', 'This user is deleted from the lab');
```
Comment (danieleynis) [inline routes/index.js]: Actually I forgot this as well. Could you please change this to "The user has been successfully deleted from the system"

File: routes/index.js (hunk 6 above)
Comment (danieleynis) [reply routes/index.js]: You spelled "confirmed" wrong

**Comment by danieleynis** [comment]
It looks good just 2 things:
1. Fix the message changes I mentioned
2. Once I approve, please squash commits

File: routes/index.js (hunk 6 above)
Comment (BishoyHanna91) [reply routes/index.js]: thanks for catching these  errors.

**Comment by BishoyHanna91** [comment]
changes done.

**Pull request merged**